gui/utils/utils.js

```diff
@@ -72,6 +72,37 @@ export const downloadFile = (fileId) => {
   }
 };
 
+
+export const deleteFile = (fileId) => {
```

Comment:
remove this function from utils.js, move the logic to ResourceManager.js and the endpoint to DashboardService.js. Have a look at DashboardService.js to get an idea on how the api integration is done